From: Erik de Castro Lopo <erikd@mega-nerd.com>
Date: Fri, 13 Jan 2017 10:30:55 +0000 (+1100)
Subject: libFLAC/cpu.c: Fix build issue on OpenBSD/i386
X-Git-Tag: 1.3.3~128
X-Git-Url: http://git.xiph.org/?p=flac.git;a=commitdiff_plain;h=6ecd1a2607a337638adb5caa67a61e7719e0e3a4

libFLAC/cpu.c: Fix build issue on OpenBSD/i386

Patch-from: Christian Weisgerber <naddy@mips.inka.de>
---

diff --git a/src/libFLAC/cpu.c b/src/libFLAC/cpu.c
index 808d55d7..12d46191 100644
--- a/src/libFLAC/cpu.c
+++ b/src/libFLAC/cpu.c
@@ -65,7 +65,7 @@ static const unsigned FLAC__CPUINFO_IA32_CPUID_SSE = 0x02000000;
 static const unsigned FLAC__CPUINFO_IA32_CPUID_SSE2 = 0x04000000;
 #endif
 
-#if FLAC__HAS_X86INTRIN || FLAC__AVX_SUPPORTED
+#if FLAC__HAS_X86INTRIN || FLAC__AVX_SUPPORTED || defined FLAC__HAS_NASM
 /* these are flags in ECX of CPUID AX=00000001 */
 static const unsigned FLAC__CPUINFO_IA32_CPUID_SSE3 = 0x00000001;
 static const unsigned FLAC__CPUINFO_IA32_CPUID_SSSE3 = 0x00000200;
